packages/dds/sequence/src/intervals/sequenceInterval.ts

```diff
@@ -529,35 +520,45 @@ export class SequenceIntervalClass implements SequenceInterval, ISerializableInt
 
 		let startRef = this.start;
 		if (startPos !== undefined) {
-			startRef = createPositionReference(
-				this.client,
+			const slidingPreference = startReferenceSlidingPreference(
 				startPos,
-				getRefType(this.start.refType),
+				startSide ?? Side.Before,
+				endPos,
+				endSide ?? Side.Before,
+			);
+			startRef = createPositionReference({
+				client: this.client,
+				pos: startPos,
+				refType: getRefType(this.start.refType),
 				op,
-				undefined,
 				localSeq,
-				startReferenceSlidingPreference(stickiness),
-				startReferenceSlidingPreference(stickiness) === SlidingPreference.BACKWARD,
-				useNewSlidingBehavior,
-			);
+				slidingPreference,
+				canSlideToEndpoint:
```

Comment:
why the `&& slidingPreference === SlidingPreference.BACKWARD` here? If a start ref prefers to slide forward, it means it is being interpreted inclusively by the application. If it reaches the end of the string and starts sliding backward because `canSlideToEndpoint` was false, this is undesirable behavior.

It's definitely an oddity of our model that start points of intervals can slide past end points, but allowing the start ref to slide to the endpoint of the sequence (and similarly for end point of a ref to the startpoint of the sequence) is consistent with that choice/oddity.

Reply:
`canSlideToEndpoint` is also a bit odd, in that is both a feature flag, and determined by the stickiness sliding preference